src/lib/auth/auth.ts

```diff
@@ -124,6 +124,9 @@ export class Auth {
   hasOrganizationAdminPermission = (orgNr: string) =>
     this.hasOrganizationRole({ orgNr, role: 'admin' });
 
+  hasOrganizationPermission = (orgNr: string) =>
```

Comment:
Denne er litt uklar. read og write er permission, admin er en rolle og ikke en permission i seg selv. Det skulle da holde å sjekke om write permission eller readpermission.

Reply:
Ja, en som har admin-rettigheter vil vel alltid ha write, og en med write vil alltid ha read. Kan droppe denne metoden og bruke readpermission i stedet :+1:

Reply:
Alternativt kalle denne for noe som f.eks. 'hasOrganizationRole'.

Reply:
Too late: 32ffa1d